src/View/Helper/ListsHelper.php

```diff
@@ -337,9 +337,17 @@ public function displayVisibilityOption($listId, $value)
             ?>
             <input type="radio"  name="visibility" data-list-id='<?= $listId ?>'  value="{{visibility}}" checked hidden ng-init="visibility = '<?= $value ?>';"/>
             <md-radio-group ng-controller='optionsCtrl' ng-model='visibility' ng-change='visibilityChanged()'>
-                <md-radio-button value='public' class='md-primary'>
+                <md-radio-button value='public+' class='md-primary' title='The list will be shown on the sentences page'>
                     <?php /* @translators: visibility option of a list */ ?>
-                    <?=  __('Public') ?>
+                    <?=  __('Public+'); 
+                    echo $this->Html->image(IMG_PATH .'new1.png');
+                    ?>
+                </md-radio-button>
+                <md-radio-button value='public' class='md-primary' title='The list can be found only over Browse -> Browse by list, not at the sentence page'>
```

Comment:
The text inside the `title` attributes is in English and cannot be displayed into other languages (when one selects a different UI language using the top-right dropdown). To make it translatable, use:
```html
<md-radio-button ... title='<?= h(__("blah blah blah")) ?>'>
```

The [`h()`](https://book.cakephp.org/3/en/core-libraries/global-constants-and-functions.html#h) function is a convenience wrapper for PHP’s `htmlspecialchars()`.